exonum/src/api/mod.rs

```diff
@@ -218,14 +220,25 @@ impl ServiceApiScope {
 /// ```
 #[derive(Debug, Clone, Default)]
 pub struct ServiceApiBuilder {
+    blockchain: Option<Blockchain>,
```

Comment:
Any specific reason for `Option` here? Why not storing just `Blockchain`? (with `ServiceApiBuilder::new(b: Blockchain)`)

Reply:
@vitvakatu Yes. The reason is to leave a method `new` as is.

Reply:
Ah okay.
@DarkEld3r I think the we might need patch release from 0.9 with this change. It's not a requirement, but it would be much simpler and cleaner solution for EJB team. Is it possible? Otherwise we need to either use git dependency, or wait till 0.10.

Reply:
@vitvakatu  It is definitely possible. :+1: